sklearn/tests/test_calibration.py

```diff
@@ -533,9 +533,11 @@ def text_data_pipeline(text_data):
 
 
 def test_calibration_pipeline(text_data, text_data_pipeline):
-    # Test that calibration works in prefit pipeline with transformer,
-    # where `X` is not array-like, sparse matrix or dataframe at the start.
-    # See https://github.com/scikit-learn/scikit-learn/issues/8710
+    """Test that calibration works in prefit pipeline with transformer,
+    where `X` is not array-like, sparse matrix or dataframe at the start.
+    See https://github.com/scikit-learn/scikit-learn/issues/8710
+    Test is can also predict without running into validation errors.
+    See https://github.com/scikit-learn/scikit-learn/issues/19637"""
```

Comment:
Nitpick: PEP 257.

```suggestion
    """Test that calibration works in prefit pipeline with transformer

    `X` is not array-like, sparse matrix or dataframe at the start.
    See https://github.com/scikit-learn/scikit-learn/issues/8710

    Also test it can predict without running into validation errors.
    See https://github.com/scikit-learn/scikit-learn/issues/19637
    """
```